### PR TITLE
Use improved context api

### PR DESCRIPTION
**Pull request by rainemak**
Requires: https://git.merproject.org/mer-core/qtmozembed/merge_requests/18

### CODE REVIEW

**Comment by rainemak** [comment]
@chriadam review ?

**Comment by chriadam** [comment]
LGTM, haven't tested (broke my device, need to reflash...)

**Comment by rainemak** [comment]
See also: https://github.com/sailfishos/sailfish-components-webview/pull/7

File: src/browser/declarativewebutils.cpp [hunk 1]
```diff
@@ -143,81 +116,48 @@ void DeclarativeWebUtils::handleDumpMemoryInfoRequest(QString fileName)
...
 
-    mozContext->setPref(QString("geo.wifi.scan"), QVariant(false));
-    mozContext->setPref(QString("browser.enable_automatic_image_resizing"), QVariant(true));
+    webEngineSettings->setPreference(QString("security.alternate_certificate_error_page"), QString("certerror"));
 
-    // Make long press timeout equal to the one in Qt
-    mozContext->setPref(QString("ui.click_hold_context_menus.delay"), QVariant(PressAndHoldDelay));
-    mozContext->setPref(QString("apz.fling_stopped_threshold"), QString("0.13f"));
+    webEngineSettings->setPreference(QString("geo.wifi.scan"), QVariant(false));
+    webEngineSettings->setPreference(QString("browser.enable_automatic_image_resizing"), QVariant(true));
```
Comment (chriadam) [inline src/browser/declarativewebutils.cpp]: Is this setting something which should be added to the WebEngine defaults?

File: tests/auto/tst_declarativewebcontainer/tst_declarativewebcontainer.cpp [hunk 2]
```diff
@@ -56,13 +56,13 @@ void tst_declarativewebcontainer::initTestCase()
 
 void tst_declarativewebcontainer::cleanupTestCase()
 {
-    delete QMozContext::GetInstance();
+    delete QMozContext::instance();
```
Comment (chriadam) [inline tests/auto/tst_declarativewebcontainer/tst_declarativewebcontainer.cpp]: Is this intentional?

Comment (chriadam) [inline tests/auto/tst_declarativewebcontainer/tst_declarativewebcontainer.cpp]: Oh, I guess the QMozContext API changed to instance().  Never mind.

**Comment by chriadam** [comment]
One comment, but LGTM

File: src/browser/declarativewebutils.cpp (hunk 1 above)
Comment (rainemak) [inline src/browser/declarativewebutils.cpp]: Good catch. Should be part of WebEngine defaults.

**Pull request merged**